.github/CONTRIBUTING.md

```diff
@@ -295,37 +295,37 @@ To setup Development Environment without Docker you need following pre-requisite
 eg. on MacOS you can follow this instructions:
 
 ```shell
-#1 install brew (https://brew.sh)
+# 1 Install Homebrew (https://brew.sh)
 /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
 
-#2 install nvm (https://github.com/nvm-sh/nvm#troubleshooting-on-macos)
+# 2 Install nvm (https://github.com/nvm-sh/nvm#troubleshooting-on-macos)
 brew update
 brew install nvm
 
-#3 install specified node version using NVM (https://github.com/nvm-sh/nvm)
+# 3 Install specified node version using NVM (https://github.com/nvm-sh/nvm)
 
 nvm install v20.8.0
 nvm alias default v20.8.0
 
-#4 install postgres (https://wiki.postgresql.org/wiki/Homebrew)
+# 4 Install postgres (https://wiki.postgresql.org/wiki/Homebrew)
 brew install postgresql@14
 brew services start postgresql@14
 
-#5 install dbt (https://docs.getdbt.com/dbt-cli/install/homebrew)
-brew tap dbt-labs/dbt@1.4.9
-brew install dbt-postgres@1.4.9
+# 5 Install dbt (https://docs.getdbt.com/dbt-cli/install/homebrew)
+brew tap dbt-labs/dbt
+brew install dbt-postgres@1.5.4

```

Comment:
`1.4.9` isn't available on Homebrew; moved to the latest `1.5.*` version available.